src/test/suite/fixtures/recorded/navigationMap/bringPointAfterFirstCarWhaleTakeWhale.yml

```diff
@@ -0,0 +1,44 @@
+spokenForm: bring point after first car whale take whale
+languageId: plaintext
+command:
+  actionName: replaceWithTarget
+  partialTargets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: .}
+    - type: primitive
+      selectionType: token
+      modifier: {type: subpiece, pieceType: character, anchor: 0, active: 0, excludeAnchor: false, excludeActive: false}
+      mark: {type: decoratedSymbol, symbolColor: default, character: w}
+      position: after
+  extraArgs: []
+initialState:
+  documentContents: hello. world whatever
+  selections:
+    - anchor: {line: 0, character: 13}
+      active: {line: 0, character: 13}
+  marks:
+    default..:
+      start: {line: 0, character: 5}
+      end: {line: 0, character: 6}
+    default.w:
+      start: {line: 0, character: 7}
+      end: {line: 0, character: 12}
+finalState:
+  documentContents: hello. w.orld whatever
+  selections:
+    - anchor: {line: 0, character: 14}
+      active: {line: 0, character: 14}
+  marks:
+    default..:
+      start: {line: 0, character: 5}
+      end: {line: 0, character: 6}
+    default.w:
+      start: {line: 0, character: 7}
+      end: {line: 0, character: 13}
```

Comment:
Notice how we just include the `.` when it's in the middle, even though it technically is breaking up the token